src/commands/utils/clusters.ts

```diff
@@ -352,4 +352,18 @@ export async function getWindowsNodePoolKubernetesVersions(
 export function getContainerClient(target: AksClusterTreeItem): azcs.ContainerServiceClient {
     const environment = target.subscription.environment;
     return new azcs.ContainerServiceClient(target.subscription.credentials, target.subscription.subscriptionId, {endpoint: environment.resourceManagerEndpointUrl});
+}
+
+export async function deleteCluster(
+    target: AksClusterTreeItem,
+    clusterName: string
+): Promise<Errorable<string>> {
+    try {
+        const containerClient = getContainerClient(target);
+        await containerClient.managedClusters.beginDeleteAndWait(target.resourceGroupName, clusterName)
+
+        return { succeeded: true, result: "Delete cluster succeeded." };
+    } catch (ex) {
+        return { succeeded: false, error: `Error invoking ${clusterName} managed cluster: ${ex}` };
```

Comment:
One thing I noticed recently: the `errorable` module exposes a `getErrorMessage` function that can be used for displaying errors in strings. Shall we start making more use of that?
```javascript
return { succeeded: false, error: `Error invoking ${clusterName} managed cluster: ${getErrorMessage(ex)}` };
```